packages/venia-ui/lib/components/SortedByContainer/sortedByContainer.css

```diff
@@ -1,12 +1,12 @@
 .root {
-    color: rgb(var(--venia-global-color-text-alt));
+    color: rgb(var(--venia-global-color-gray-700));
     text-align: center;
     font-size: 0.875rem;
     padding-bottom: 1rem;
 }
 
 .sortText {
-    font-weight: 600;
+    font-weight: 700;
```

Comment:
was this intentional?

Reply:
Yes, text in 600 weight was slightly hard to differentiate example below (top is 600, bottom is 700): 

![screenshot1](https://user-images.githubusercontent.com/36369921/135317965-d7847991-1ca5-4d56-a6dd-c4a7a4a64b23.png)
![screenshot2](https://user-images.githubusercontent.com/36369921/135317972-71f08cfd-44b5-4092-a68b-e9f18fbd6706.png)

Reply:
Yes, sorry. I'll undo this.